frames/src/main/java/com/checkout/frames/screen/manager/PaymentFormStateManager.kt

```diff
@@ -25,7 +26,7 @@ internal class PaymentFormStateManager(
     override val cvv: MutableStateFlow<String> = MutableStateFlow("")
     override val isCvvValid: MutableStateFlow<Boolean> = MutableStateFlow(false)
 
-    override val cardHolderName: MutableStateFlow<String> = MutableStateFlow("")
+    override val cardHolderName = MutableStateFlow(paymentFormCardHolderName.ifEmpty { "" })
```

Comment:
hmmm is it the same as this?

```suggestion
    override val cardHolderName = MutableStateFlow(paymentFormCardHolderName)
```